host/volume/backend.go

```diff
@@ -17,6 +18,12 @@ type Provider interface {
 
 	NewVolume() (Volume, error)
 	DestroyVolume(Volume) error
+	CreateSnapshot(Volume) (Volume, error)
+	ForkVolume(Volume) (Volume, error)
+
+	ReportHaves(Volume) ([]json.RawMessage, error) // Report known data addresses; this can be given to `SendSnapshot` to attempt deduplicated/incrememntal transport.
```

Comment:
Renamed function.

Maybe a type aliasing `json.RawMessage` with a more local name would also improve clarity, thoughts?  I don't think it would actually give us much more for static type checking, so I'm on the fence.